service/s3/s3manager/download.go

```diff
@@ -50,6 +50,9 @@ type Downloader struct {
 	// List of request options that will be passed down to individual API
 	// operation requests made by the downloader.
 	RequestOptions []request.Option
+
+	// Strategy for buffering writes for downloaded parts
+	DownloadStrategy WriterReadFromProvider
```

Comment:
Following the `Uploader`'s pattern this member should be named, `BufferProvider`